transports_test.go

```diff
@@ -0,0 +1,12 @@
+package image
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKnownTransports(t *testing.T) {
+	assert.NotNil(t, KnownTransports) // Ensure that the initialization has actually been run
+	assert.True(t, len(KnownTransports) > 1)
```

Comment:
(Surprisingly enough, I have actually managed to write the initialization incorrectly the first time. Hence this test.)